backend/hitas/views/apartment.py

```diff
@@ -813,6 +813,12 @@ def download_latest_unconfirmed_prices(self, request, **kwargs) -> Union[HttpRes
             )
 
         sapc = SurfaceAreaPriceCeiling.objects.only("value").get(month=this_month())
+        apartment_data["prices"]["acquisition_price"] = (
+            apartment_data["prices"]["debt_free_purchase_price"] + apartment_data["prices"]["primary_loan_amount"]
+            if apartment_data["prices"]["debt_free_purchase_price"] is not None
+            and apartment_data["prices"]["primary_loan_amount"] is not None
+            else None
+        )
```

Comment:
This same thing is already done in the `PricesSerializer`, so why is this here? 
The value of `acquisition_price` will never be changed by this

Reply:
Somehow didn't see this in the serializer, must have been sleeping on the job. Anyway, removed this as it is unnecessary.